storage/postgres/tree_storage_test.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2018 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package postgres
+
+import (
+	"strings"
+	"testing"
+)
+
+type expandTestcase struct {
+	input    *statementSkeleton
+	expected string
+}
+
+func TestInitializes(t *testing.T) {
+	_ = &statementSkeleton{}
```

Comment:
Sorry, I feel like I'm missing something obvious, but other than the call to `newTreeStorage`, what are the rest of these assignment statements testing?

Reply:
This was just to get around gometalinter's megacheck that functions/structs were going unused

Reply:
Aha, ok - maybe pop a comment in there for other folks too? :)